recipes/biobear/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "biobear" %}
+{% set version = "0.7.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biobear-{{ version }}.tar.gz
+  sha256: 6fb47bee0ff81d99a687ffd316e1be5549a4653c1b8ff1c6b7e4d500bea5b228
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - maturin >=1.1.0,<2.0.0                 # [build_platform != target_platform]
+
+  host:
+    - python
+    - maturin >=1.1.0,<2.0.0
+    - pip
+    - xz                                     # [linux]
```

Comment:
Why is this dependency Linux only? Any problems on macOS/Windows that we can help?

Reply:
Hmmm... it only seemed to be needed to compile on linux, but perhaps I should make it for all platforms?